cmd/create.go

```diff
@@ -247,12 +249,22 @@ func newCreateConfig(cmd *cobra.Command, args []string, newClient ClientFactory)
 		Confirm:          viper.GetBool("confirm"),
 		Verbose:          viper.GetBool("verbose"),
 	}
+
+	// Using Visit func to count the number of flags set at prompt. If 0 then set confirm mode to true.
```

Comment:
At the moment the only mandatory flag is the one for language (`-l`) shouldn't we check the presence of this instead?

if user runs `func create -t http my-func` the confirm mode will not be enabled, but the language will not be selected.

Reply:
@lance any comments on that? I've just implemented the requested functionality "if `pure func create` then `run using confirm mode`"...

Reply:
This is... a little opaque.
While I admire the cleverness of "if no flags, default to confirm"; I think this will end up causing more confusion than clarity